apps/editors/views.py

```diff
@@ -222,21 +222,29 @@ def performance(request, user_id=False):
     months = ReviewerScore.get_breakdown_since(user, month_ago)
     years = ReviewerScore.get_breakdown_since(user, year_ago)
 
-    def _sum(iter, types):
-        return sum(s.total for s in iter if s.atype in types)
+    def _sum(iter, types, exclude=False):
+        return sum(s.total
+                   for s in iter
+                   if (s.atype in types) == (not exclude))
```

Comment:
Added a docstring. I'm not crazy about the wording but found it a bit weird to describe.